start.py

```diff
@@ -0,0 +1,121 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import yaml
+from socket import getfqdn, socket
+from time import sleep, time
+
+from clusterdock.models import Cluster, client, Node
+from clusterdock.utils import wait_for_condition
+
+logger = logging.getLogger('clusterdock.{}'.format(__name__))
```

Comment:
Move defining `logger` to be the last thing before `def main`. Also, run `flake8` against this whole repo to find other style issues before submitting the next iteration of this commit.

Reply:
Done. Ran with flake8.